cconv.c

```diff
@@ -237,8 +237,8 @@ void convert_thread_options_to_cpu(struct thread_options *o,
 	o->loops = le32_to_cpu(top->loops);
 	o->mem_type = le32_to_cpu(top->mem_type);
 	o->mem_align = le32_to_cpu(top->mem_align);
-	o->exit_what = le16_to_cpu(top->exit_what);
```

Comment:
Pretty sure this breaks alignment and compilation will fail...

Reply:
Yes, I've now removed one of the manual paddings to get it aligned again.